### PR TITLE
#167750043 Setup Test with Mocha, Chai and Chai-http

### PR DESCRIPTION
**Pull request by Oliver-ke**
#### Description of Task to be completed?
- Install mocha, chai and chai-http
- add test folder in src
- create a simple basic test file
- add test script to package.json file

 #### How should this be manually tested?
- open the command prompt
- run `npm install -D mocha chai chai-http` to install dev dependencies
- run  ` npm run test ` to start-up the test suite

 #### Any background context you want to provide?
None

 #### What are the relevant pivotal tracker stories?
#[167750043](https://www.pivotaltracker.com/story/show/167750043)

 #### Screenshots (if appropriate)
None
 #### Questions:
None

### CODE REVIEW

**Comment by sa-ma** [comment]
Great job @Oliver-ke  just one little observation. We only need to run `npm install` not installing the modules separately, that way it ensures we install all the modules specified in package.json

**Review by sa-ma — DISMISSED** [review]

**Review by feobaby — DISMISSED** [review]
Nice work here, totally fine with it.

File: package.json [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
+	"scripts": {
+		"test": "cross-env NODE_ENV=test && mocha src/test --require @babel/register --recursive --exit || true",
+		"start:dev": "cross-env NODE_ENV=development nodemon --exec babel-node ./src/index.js"
+	},
+	"author": "Andela Simulations Programme",
+	"license": "MIT",
+	"dependencies": {
+		"body-parser": "^1.18.3",
+		"chai": "^4.2.0",
+		"chai-http": "^4.3.0",
```
Comment (eokwukwe) [inline package.json]: `chai`, `chai-http`, and `mocha` should be in `devDependencies`

**Review by eokwukwe — CHANGES_REQUESTED** [review]
@Oliver-ke. Check the title of your PR and confirm that it is in the correct format.

File: package.json (hunk 1 above)
Comment (Oliver-ke) [reply package.json]: Thanks, @eokwukwe , Those requested changes have been done

**Comment by torsami77** [comment]
@Oliver-ke, Great Job, the brace brackets surrounding the story ID in your PR title is contrary to the convention, you might want to check it again

**Review by torsami77 — CHANGES_REQUESTED** [review]
@Oliver-ke, Great Job, the brace brackets surrounding the story ID in your PR title is contrary to the convention, you might want to check it again

**Review by sa-ma — APPROVED** [review]

**Review by feobaby — APPROVED** [review]

**Review by eokwukwe — APPROVED** [review]

**Pull request merged**